walkthroughs/week-2-server/examples/random-quotes/src/main/java/com/google/sps/servlets/RandomQuoteServlet.java

```diff
@@ -52,7 +52,8 @@ public void init() {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String quote = quotes.get((int) (Math.random() * quotes.size()));
 
-    response.setContentType("text/html;");
+    response.set
```

Comment:
Looks like this is in the examples anyway, but I'm guessing this newline is just a typo?